el8/Dockerfile.buildtime

```diff
@@ -0,0 +1,18 @@
+FROM @BASE_IMAGE_NAME@
+LABEL maintainer="CMS Build"
+LABEL name="CMS Worker Node on EL - Buildtime"
```

Comment:
@aandvalenzuela , this worked ... right? If yes then `LABEL build-date="@BUILD_DATE@"` should also work

Reply:
Right, I think I found what I was doing wrong. I am pushing the changes soon